specification/windowsesu/resource-manager/Microsoft.WindowsESU/preview/2019-09-16-preview/windowsesu.json

```diff
@@ -0,0 +1,571 @@
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-long-running-operation": true
+      },
+      "patch": {
+        "tags": ["MultipleActivationKeys"],
+        "operationId": "MultipleActivationKeys_Update",
+        "x-ms-examples": {
+          "UpdateMultipleActivationKey": { "$ref": "./examples/UpdateMultipleActivationKey.json" }
+        },
+        "description": "Update a MAK key.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/MultipleActivationKeyNameParameter"
+          },
+          {
+            "name": "multipleActivationKey",
+            "in": "body",
+            "description": "Details of the MAK key.",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/MultipleActivationKeyUpdate"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/MultipleActivationKey"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        }
+      },
+      "delete": {
+        "tags": ["MultipleActivationKeys"],
+        "operationId": "MultipleActivationKeys_Delete",
+        "x-ms-examples": {
+          "DeleteMultipleActivationKey": { "$ref": "./examples/DeleteMultipleActivationKey.json" }
+        },
+        "description": "Delete a MAK key.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/MultipleActivationKeyNameParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK"
+          },
+          "204": {
+            "description": "No Content"
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "MultipleActivationKey": {
+      "description": "MAK key details.",
+      "type": "object",
+      "required": [
+        "location"
+      ],
+      "properties": {
+        "id": {
```

Comment:
It's best to carve the common resource properties into a shared definition. Better yet, use an allOf reference to link the common definition from common-types. https://github.com/Azure/azure-rest-api-specs/blob/a6f41b54a296e11631a65bbe4a2da623afb758a2/specification/common-types/resource-management/v1/types.json#L45

Reply:
I've added references to common types